tstest/main.ts

```diff
@@ -12,8 +12,31 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
-import {connect} from '..'
+import {
+    connect,
+    BAD_AUTHENTICATION,
```

Comment:
Should we put these under something so we can pull them all in with one import?

Reply:
for typescript each export is imported.

Reply:
But could we import "CODES" and do CODES.<SOMETHING>?

Reply:
ok I see this would have to be a 2.0 thing..